backend/src/test/java/com/team4/backend/testdata/InternshipOfferMockData.java

```diff
@@ -66,6 +100,52 @@ public static InternshipOffer getInternshipOffer() {
                         .build());
     }
 
+    public static List<Date> getSessionDates() {
```

Comment:
im feeling like this is the same code as in the service, like you are repeating yourself (just a remark)